packages/flutter_tools/lib/src/migrations/swift_package_manager_integration_migration.dart

```diff
@@ -529,7 +586,7 @@ $newContent
       );
       if (startFilesIndex == -1 || startFilesIndex > endSectionIndex) {
         throw Exception(
-          'Unable to files for PBXFrameworksBuildPhase ${_xcodeProject.hostAppProjectName} target.',
+          'Unable to find files for PBXFrameworksBuildPhase ${_xcodeProject.hostAppProjectName} project.',
```

Comment:
This is presumably a typo. Unfortunately this line isn't already tested. However, the if-check on line 574 would technically make this line unreachable?